ols/src/ui/gradio_ui.py

```diff
@@ -39,7 +39,7 @@ def __init__(
     def chat_ui(
         self,
         prompt: str,
-        history,
+        history: list[list[Optional[str]]],
```

Comment:
this argument isn't even used, how did you conclude this should be its data type?

(I think we should just be deleting the argument)

Reply:
is not `prompt` and `history` used automatically by the Chat widget?

Reply:
and the conclusion - I looked into Gradio's sources https://github.com/gradio-app/gradio/blob/7302a6e151dac553c17833be64d4639ee4cf97aa/gradio/chat_interface.py

Reply:
sorry, you're correct:

from the gradio docs:

```
Should accept two parameters: a string input message and list of two-element lists of the form [[user_message, bot_message], ...] representing the chat history
```